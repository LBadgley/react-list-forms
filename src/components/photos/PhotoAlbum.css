.PhotoAlbum {
  width: 100rem;
  padding: 1rem;
  margin: auto;

  h2 {
    padding: 20px;

    font-size: 2rem;
    text-align: center;
    text-decoration: underline;
  }

  ul {
    display: flex;
    justify-content: space-around;
    align-items: center;
    list-style-type: none;
  }

  li {
    height: 10rem;
  }

  img {
    width: auto;
    height: 100%
  }
}
