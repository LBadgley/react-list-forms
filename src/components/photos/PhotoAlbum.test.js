import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('photo album test', () => {
  it('photo album renders', () => {
    const photos = [
      '../../assets/calico.jpg',
      '../../assets/fatGreyTabby.jpg',
      '../../assets/nebelung.jpg'
    ];
    const wrapper = shallow(<PhotoAlbum title="Family Photos" photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
