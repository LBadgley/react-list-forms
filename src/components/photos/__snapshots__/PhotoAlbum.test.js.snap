// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photo album test photo album renders 1`] = `
<section
  className="PhotoAlbum"
>
  <h2>
    Family Photos
  </h2>
  <[object Object]
    photos={
      Array [
        "../../assets/calico.jpg",
        "../../assets/fatGreyTabby.jpg",
        "../../assets/nebelung.jpg",
      ]
    }
  />
</section>
`;
